### PR TITLE
Get single menu - addon and category population

### PR DESCRIPTION
**Pull request by lazykern**

### CODE REVIEW

File: src/menus/menus.service.ts [hunk 1]
```diff
@@ -172,10 +170,10 @@ export class MenusService {
 
   async deleteManyMenus(ids: Types.ObjectId[]) {
     const currentDate = new Date();
-    const deleteManyScript = {
+    const deleteManyAggregation = {
```
Comment (ishiasaka) [inline src/menus/menus.service.ts]: It's not aggregation. please change to query.

Comment (lazykern) [reply src/menus/menus.service.ts]: k bruh

Comment (lazykern) [reply src/menus/menus.service.ts]: mb

**Review by ishiasaka — APPROVED** [review]

**Comment by lazykern** [comment]
image

**Pull request merged**